saleor/cart/test_cart.py

```diff
@@ -77,8 +18,8 @@ def get_variant(self, cleaned_data):
         return self.product
 
 
-stocked_variant = BigShipVariant(name='Big Ship')
-non_stocked_variant = ShipPhotoVariant(name='Ship Photo')
+stocked_variant = ProductVariant(name='Big Ship')
+non_stocked_variant = ProductVariant(name='Ship Photo')
```

Comment:
This does not make sense now. These two are identical, both of them are stocked.